### PR TITLE
Fix parse_url expects string, resource given error

### PR DESCRIPTION
**Pull request by mikemeier**
When I work with resources, I got the error: Warning: parse_url() expects string, resource given.

### CODE REVIEW

**Comment by mikemeier** [comment]
@aztech-dev Is this PR ok or should I change anything?

File: src/Resource/Resource.php [hunk 1]
```diff
@@ -101,6 +101,10 @@ public function getContextForProcessInSinglePlace()
      */
     private function isLocal()
     {
+        if(is_resource($this->original)){
```
Comment (aztech-dev) [inline src/Resource/Resource.php]: To keep in line with the library's coding style, it should be `if (is_resource($this->original)) {`

**Comment by aztech-dev** [comment]
Sorry I missed your P/R. I left a comment about coding style, additionnally, do you mind adding a unit test that matches your scenario where you got that error ?

Thanks !

**Comment by mikemeier** [comment]
@aztech-dev Ok I added a test and formatted the code correct

**Comment by aztech-dev** [comment]
Thanks a lot ! Once the tests are done running (and green) I'll merge your P/R.

**Comment by mikemeier** [comment]
@aztech-dev Will you also create a new release/tag? Thanks!

**Comment by aztech-dev** [comment]
Working on it, I need to document all the changes though, especially considering it's a BC release

**Pull request merged**